core/storage/user/gae_models.py

```diff
@@ -152,6 +152,9 @@ class UserStatsModel(base_models.BaseMapReduceBatchResultsModel):
     The impact score is 0 for an exploration with 0 playthroughs or with an
     average rating of less than 2.5.
     """
+    # This value should be updated in the event of any event schema change.
+    CURRENT_EVENT_SCHEMA_VERSION = 1
```

Comment:
There is this constant, and there is feconf.CURRENT_DASHBOARD_STATS_SCHEMA_VERSION. It's bad practice to have two constants tracking the same thing -- you're just asking for skew in the future. I would get rid of this and point to the constant in feconf instead.